cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/ConsentStatusApiModel.kt

```diff
@@ -63,19 +63,35 @@ data class ConsentStatusResp(
 
 @Serializable
 data class CcpaCS(
-    @SerialName("applies") val applies: Boolean?,
-    @SerialName("ccpaApplies") val ccpaApplies: Boolean?,
-    @SerialName("consentedAll") val consentedAll: Boolean?,
-    @SerialName("dateCreated") val dateCreated: String?,
-    @SerialName("gpcEnabled") val gpcEnabled: Boolean?,
-    @SerialName("newUser") val newUser: Boolean?,
-    @SerialName("rejectedAll") val rejectedAll: Boolean?,
-    @SerialName("rejectedCategories") val rejectedCategories: List<String>?,
-    @SerialName("rejectedVendors") val rejectedVendors: List<String>?,
-    @SerialName("signedLspa") val signedLspa: Boolean?,
-    @Serializable(with = CcpaStatusSerializer::class) val status: CcpaStatus?,
-    @SerialName("uuid") var uuid: String?,
-    @SerialName("webConsentPayload") val webConsentPayload: JsonObject? = null,
+    @SerialName("applies")
+    val applies: Boolean?,
+    @SerialName("ccpaApplies")
+    val ccpaApplies: Boolean?,
+    @SerialName("consentedAll")
+    val consentedAll: Boolean?,
+    @SerialName("dateCreated")
+    val dateCreated: String?,
+    @SerialName("gpcEnabled")
+    val gpcEnabled: Boolean?,
+    @SerialName("newUser")
+    val newUser: Boolean?,
+    @SerialName("rejectedAll")
+    val rejectedAll: Boolean?,
+    @SerialName("rejectedCategories")
+    val rejectedCategories: List<String>?,
+    @SerialName("rejectedVendors")
+    val rejectedVendors: List<String>?,
+    @SerialName("signedLspa")
+    val signedLspa: Boolean?,
+    @Serializable(with = CcpaStatusSerializer::class)
+    val status: CcpaStatus?,
```

Comment:
`status` doesn't have a `@SerialName("status")` is it intentional?

Reply:
yes, this line was not changed, and the serializer will use the name of the parameter (i.e. `"status"`)